f5lbaasdriver/v2/bigip/driver_v2.py

```diff
@@ -68,6 +70,12 @@ def __str__(self):
 
 class F5DriverV2(object):
     u"""F5 Networks® LBaaSv2 Driver."""
+    _rpc_warning = """"Please note that by having the parent own the RPC worker
```

Comment:
Line 73 is not a doc string.  It's a LOG.warning message that is logged if/when a customer decides to put workers to 1 or lower.  Would you still like this message to be tabbed?  Also, please let me know if you would prefer this to be handled elsewhere or in a different manner.  I put it here because if I were a user that ran into this as a problem, I would prefer it to be open and transparent as I've got here, but being new, it may be more appropriate elsewhere that I'm not aware of.